src/js/webSerial.js

```diff
@@ -30,12 +30,32 @@ class WebSerial extends EventTarget {
 
         this.logHead = "SERIAL: ";
 
+        this.port_counter = 0;
+        this.ports = null;
```

Comment:
```suggestion
        this.ports = [];
```

Reply:
I thought the same, but I need then to add another variable to "detect" if the ports have been populated or simply there isn't any. But I will give it a look to do in a better way.

Reply:
Fixed.